src/internal/InnerSubscriber.ts

```diff
@@ -3,13 +3,14 @@ import { OuterSubscriber } from './OuterSubscriber';
 
 /**
  * We need this JSDoc comment for affecting ESDoc.
+ * @internal
  * @ignore
  * @extends {Ignored}
  */
 export class InnerSubscriber<T, R> extends Subscriber<R> {
-  private index: number = 0;
+  private index = 0;
 
-  constructor(private parent: OuterSubscriber<T, R>, private outerValue: T, private outerIndex: number) {
+  constructor(private parent: OuterSubscriber<T, R>, public outerValue: T, public outerIndex: number) {
```

Comment:
Double checking that going from private to public was desired. Was this changed for tests that aren't in this PR that stopped passing because they accessed these, but passed previously?

Reply:
Hmm... I remember making this change as I worked on this incrementally, because there were some ineffeciencies going on around passing these two things around. Given that `OuterSubscriber` is really an internal-only type, there's probably not an issue here. But if it's not being used in this PR, I can revert that part.

Reply:
Agreed. Your call. 👍